__tests__/api/topics/follow.test.js

```diff
@@ -69,7 +71,7 @@ describe('PUT /topics/follow', () => {
     // Assertion
     expect(response.statusCode).toBe(200);
     expect(response.body.message).toBe('Success delete topic user');
-  });
+  }); */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, consider removing deprecated code instead of commenting it out.

```diff
-  }); */
```

Please ensure that the deletion of this test case does not affect the overall test coverage significantly. If it does, you might need to add new tests to maintain the coverage.


<!-- This is an auto-generated comment by OSS CodeRabbit -->